consensus/src/round_manager.rs

```diff
@@ -66,6 +73,14 @@ impl UnverifiedEvent {
                 s.verify(validator)?;
                 VerifiedEvent::SyncInfo(s)
             }
+            UnverifiedEvent::CommitVote(cv) => {
+                cv.verify(validator)?;
+                VerifiedEvent::CommitVote(cv)
+            }
+            UnverifiedEvent::CommitDecision(cd) => {
+                cd.verify(validator)?;
```

Comment:
we can remove the verify in commit_phase after this

Reply:
resolved